src/map/skill.cpp

```diff
@@ -5582,7 +5582,7 @@ int32 skill_castend_damage_id (struct block_list* src, struct block_list *bl, ui
 
 	case RG_BACKSTAP:
 		{
-			if (!check_distance_bl(src, bl, 0)) {
+			if (check_distance_bl(src, bl, 1)) {
```

Comment:
Hmm, the problem here is that the previous check was there because if you are on the exact same cell as the target, you cannot calculate the direction to the target.

But I'm also not sure how it's supposed to work in renewal. It seems to move you to behind the target?

But how does it fail on autocast? You mean like a skill moved you on the same cell as the target and that skill is then supposed to autocast Backstab?

Reply:
Sorry small correction, Kokotewa isn't fully correct, northwest and southwest also works. I'm highlighting it here:
![grafik](https://github.com/user-attachments/assets/d4f89ef4-95a4-4898-87b8-264dc07d7973)